### ISSUE TITLE
Custom templates

### ISSUE THREAD

**Issue by JasonEtco** [liuderchi/gh-template-bot#5]
Hey @liuderchi, this is really awesome! I have a feature request:

What if you could have your own custom templates? For example, imagine typing this:

```
/template bug-report Everything is broken!
```

And it would read from a file like this, maybe stored in the repo's `.github` folder:

```md
%DESCRIPTION%

## Bug report
...
```

And fill in the `%DESCRIPTION%` with the second part of the command. What do you think?

**Comment by liuderchi** [comment]
@JasonEtco custom templates are really what I'm planning to do in next release. 👍 

 - [x] Next release I will use custom templates from `.github/`folder
    - Now I'm designing the logic of *1st part of command*

  - I have not think of *2nd part of the command*, but it's interesting and worth a more detailed discussion (*what values can be interpolated in template? how is the cmd interface? classic scenarios?*).

In summary, I would prefer do those 2 things in parallel
  - implement custom template feature, but with simply *first part of the command*
  - discuss template with interpolation (perhaps in another issue?)

**Comment by JasonEtco** [comment]
Awesome @liuderchi! I'll close this out then 🎉
